packages/stark-ui/src/modules/pretty-print/components/pretty-print.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="stark-pretty-print">
```

Comment:
The class in this element should not be added anymore since we tell Angular to add it via the @HostBinding("class").

Can you remove it and adapt the SCSS styles if needed?